omod/src/main/java/org/openmrs/module/owa/web/controller/OwaManageController.java

```diff
@@ -50,13 +50,13 @@
 		return appList;
 	}
 	
-	@RequestMapping(value = "/deleteApp", method = RequestMethod.GET)
-	public String deleteApp(@RequestParam("appName") String appName, ModelMap model) {
+  @RequestMapping(value = "/deleteApp", method = RequestMethod.GET)
```

Comment:
Why has the indentation here been changed?

Reply:
@mkiterian Thanks, i have updated the PR